plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/ImportCloudConnectorWizardPage.java

```diff
@@ -247,11 +195,11 @@ public static IProject getProject(Object obj){
 	}
 
 	private void validate() {
-		if ((getCloudConnectorPath() == null || getCloudConnectorPath().equals(""))) {
-			setErrorMessage("Please specify a connector path");
-			setPageComplete(false);
-			return;
-		}
+//		if ((getCloudConnectorPath() == null || getCloudConnectorPath().equals(""))) {
+//			setErrorMessage("Please specify a connector path");
+//			setPageComplete(false);
+//			return;
+//		}
```

Comment:
Can we remove these commented code?

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/832